components/permit-holders/DoctorInformationCard.tsx

```diff
@@ -36,12 +40,12 @@ export default function DoctorInformationCard(props: DoctorInformationProps) {
         </Box>
         <Box>
           <Text as="p" textStyle="body-regular">
-            {`MSP Number: ${physician.mspNumber}`}
+            {`MSP Number: ${physician?.mspNumber || ''}`}
```

Comment:
```suggestion
            {`MSP Number: ${physician.mspNumber || ''}`}
```
since `physician` is mandatory. Same with the instances below